### PR TITLE
Rodneysbranch

### PR DESCRIPTION
**Pull request by rnutall**
created seeds folder and a Q & A text file for the test questions & answers

### CODE REVIEW

**Comment by rnutall** [comment]
hi

File: db/schema.sql
Comment (sandragruiz) [inline db/schema.sql:1]: this file is not needed, this is already being done in the models folder

File: db/seeds.sql
Comment (sandragruiz) [inline db/seeds.sql:1]: seeds go in the seeds folder, not as a sql file, but as json files so that we can run npm seed script

File: db/server.js
Comment (sandragruiz) [inline db/server.js:1]: server js does not go under db

**Review by sandragruiz — APPROVED** [review]

**Pull request merged**